src/py/flwr/driver/app.py

```diff
@@ -208,3 +197,5 @@ def update_client_manager(
 
         # Sleep for 3 seconds
         time.sleep(3)
+    # Exit
+    del driver
```

Comment:
Why is this necessary? Doesn't this happen automatically when we exit the function?

Reply:
Double checked. It is redundant. Removed.